axelor-base/src/main/java/com/axelor/apps/base/service/AddressServiceImpl.java

```diff
@@ -341,4 +351,60 @@ public String getZipCode(Address address) {
     }
     return null;
   }
+
+  @Override
+  @Transactional(rollbackOn = {Exception.class})
+  public void setFormattedFullName(Address address) {
+        computeStringAddressTemplate(address);
+  }
+
+  @Override
+  @Transactional(rollbackOn = {Exception.class})
+  public int computeFormattedAddressForCountries(List<Long> countryIds) {
+        int totalAddressFormatted = 0;
+        AddressRepository addressRepository = Beans.get(AddressRepository.class);
```

Comment:
don't inject the AddressRepository again, just use the addressRepo global variable